hknweb/tours/views.py

```diff
@@ -0,0 +1,54 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.views import generic
+from django.core.mail import EmailMultiAlternatives
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.views.generic.edit import FormView
+from .forms import *
```

Comment:
lgtm has a good point here that this can pollute the local namespace with whatever is under the forms module, which can vary. Usually it's better to explictly specify here what functions/classes you actually need in this file and only import those